.github/workflows/lint.yaml

```diff
@@ -2,6 +2,9 @@ name: Run lint
 
 on: [ push, pull_request ]
 
+permissions:
+  contents: read
```

Comment:
I am wondering: does this remove the ability of this action to post annotations? Let's try...

Reply:
Never mind, it still works:
https://github.com/go-logr/logr/pull/184/files